modules/ROOT/pages/ch2-features.adoc

```diff
@@ -0,0 +1,281 @@
++
+CloudHub 2.0 alerts you when when applications become unresponsive.
+* Load-based cluster-level autoscaling
++
+CloudHub 2.0 provides auto-scaling groups for fault-tolerance.
+* Bursting and CPU oversubscription for application deployments
+* Application administration and supportability
+* Custom properties
++
+For information on setting custom properties, see 
+xref:ch2-manage-props.adoc[].
+* xref:ch2-deploy.adoc[].
++
+CloudHub 2.0 provides a secure cloudhub.io endpoint for sending requests to applications.
+* xref:ps-config-log-forwarding.adoc[].
+
+////
+
+== CloudHub 2.0 Feature Comparison
+
+This table compares the features and support for CloudHub 2.0, CloudHub 1.0, and Runtime Fabric Self-Managed Kubernetes.
+
+[%header,cols="20a,20a,20a,20a"]
+.Feature Comparison 
+|===
+| Capability | CloudHub 2.0 | CloudHub 1.0 | Runtime Fabric On-Premises
+4+h|VM Management
+| *Provisioning/scaling*|Fully managed | Fully managed | AWS/
+
+Self-managed
+| *OS patching* | Fully managed | Fully managed | AWS/
+
+Self-managed
+| *VM self-healing* |Fully managed | Fully managed | Supported (managed nodes)
+4+h|Load Balancing and Ingress
+| *Default endpoints*|Fully managed | Fully managed | Not supported
+| *URL rewriting* | Supported
+
+(app-level) | Supported
+
+(DLB) | Supported
+|*Provisioning/updating* |Fully-managed|Fully-managed| Supported
+|*Certificate management*|Supported|Supported|Supported
+|*Multiple Trust Stores* 
+
+*(client certs)* | Supported | Not supported |Supported
+| *Custom endpoints* | Supported | Supported | Supported
+| *Multiple custom endpoints*
+
+*per app* | Supported| Partially supported | ?
+| *TLS 1.0* | Not supported | Supported| Supported
+| *Internal endpoints* | Supported |Supported | Supported
+| *Load balancer logs* | Supported
+
+(download) | Not supported | Supported
+4+h|Network
+| *Deployment regions* | 12 geographical regions | 12 geographical regions | Supported 
+
+(manual)
+|*Virtual Private Cloud* | Fully managed
+
+(private space) | Fully managed (Anypoint VPC) | Supported 
+
+(manual)
+|*Virtual Private Network* | Fully managed
+
+(private space) | Fully managed
+
+(Anypoint VPN) | Supported 
+
+(manual)
+| *AWS Transit Gateway* | Supported
+
+(private space) | Supported | Supported
+| *Direct Connect* | Not supported | Supported
+
+(not self-serve) | Supported
+|*VPC Peering* | Not supported | Supported 
+
+(not self-serve) | Supported
+
+| *DNS* | Fully managed | Fully managed | Supported
+| *Inbound firewall rules*| Supported | Supported| Supported
+| *Outbound firewall rules*| Supported
+
+(private space) | Not supported| Supported
+|*Static IP addresses* | Supported
+
+(private space) | Supported
+
+(per app) | ?
+4+^h|App Capabilities
+|*Schedule management* | Supported | Supported | Not supported
+|*TLS certificates*
+
+*for API Gateway deployments*| Not supported | Supported | Not supported
+| *DataGraph* | Not supported | Supported | Not supported
```

Comment:
Did datagraph team mention when this will be supported? @jeff-1amstudios Should we also list out partner manager, dataloader etc.?

Reply:
No change.